site/src/documents/pages/blocks/PageContentBlock.tsx

```diff
@@ -1,15 +1,16 @@
 import { BlocksBlock, PropsWithData, SupportedBlocks, YouTubeVideoBlock } from "@comet/cms-site";
```

Comment:
Does this file need `use client`? 🤔 

```suggestion
"use client";
import { BlocksBlock, PropsWithData, SupportedBlocks, YouTubeVideoBlock } from "@comet/cms-site";
```